src/vault/client.clj

```diff
@@ -70,7 +79,41 @@
   (delete-secret!
     [this path]
     (swap! memory dissoc path)
-    true))
+    true)
+
+  (create-token! [this] (create-token! this nil))
+
+  (create-token!
+    [this wrap-ttl]
+    {:request_id "",
+     :lease_id "",
+     :renewable false,
+     :lease_duration 0,
+     :data nil,
+     :wrap_info
+     {:token "87e771a4-73a5-8eda-fe59-a5cc5d41a6d2",
```

Comment:
Generate these instead of hard-coding them? You're also ignoring `wrap-ttl` here.